### ISSUE TITLE
Decision: No certification for Canvas implementation

### ISSUE THREAD

**Issue by brian-burton** [tmforum-oda/oda-canvas#219]
# New decision: No certification for canvas implementation
## Description
There will be no formal certification of a specific canvas implementation, only the components. Functional tests will be created and used to verify the canvas implementation is functioning as expected.
## Rationale
From the beginning we made the decision that components should be certified against the component specification and that a canvas must be provided as a platform that supports any certified component, the canvas itself is only the net result of ensuring the components can run and not a certifiable thing itself.
## Implication
We'll still write BDD scripts for the canvas functionality so that we can direct the functionality of the Canvas to meet the component specification. Providers of canvas services will still need to write TDD scripts to ensure their canvas services operate as expected. This includes the Innovation Hub for the reference implementation of the canvas. However, while these tests can be run in conjunction with the component compliance test kits to help ensure/diagnose the correct functioning of each canvas implementation, there will be no certification for the canvas itself.

**Comment by emmanuel-a-otchere** [comment]
Agree. The focus of certification should be on the Components. The canvas is offered to enable test the ODA Component conformance.
